### PR TITLE
rebuild.sh: Add patch to fix Time-Local tests for year 2020

### PR DESCRIPTION
**Pull request by zakame**
- https://rt.cpan.org/Public/Bug/Display.html?id=124787
- https://github.com/Perl/perl5/issues/17410

### CODE REVIEW

File: lib/5.26.2/x86_64-linux/CORE/config.h [hunk 1]
```diff
@@ -9,9 +9,9 @@
 
 /* Package name      : perl5
  * Source directory  : .
- * Configuration time: Mon Jul 30 11:50:48 UTC 2018
- * Configured by     : git
- * Target system     : linux default-debian-7 3.2.0-5-amd64 #1 smp debian 3.2.96-3 x86_64 gnulinux 
+ * Configuration time: Mon Feb  3 05:15:30 UTC 2020
+ * Configured by     : vagrant
+ * Target system     : linux debian-9 4.9.0-11-amd64 #1 smp debian 4.9.189-3+deb9u2 (2019-11-11) x86_64 gnulinux
```
Comment (zakame) [inline lib/5.26.2/x86_64-linux/CORE/config.h]: Due rebuilding this on an actual machine rather than in a Vagrant VM.

File: rebuild.sh [hunk 2]
```diff
@@ -5,6 +5,9 @@ wget http://www.cpan.org/src/5.0/perl-5.26.2.tar.gz
 tar xzvf perl-5.26.2.tar.gz
 cd perl-5.26.2
 
+# Apply patch for fixing Time::Local tests entering year 2020  (CPAN RT#124787)
+curl -fsSL https://rt.cpan.org/Public/Ticket/Attachment/1776857/956088/0001-Fix-Time-Local-tests.patch | patch -p1
```
Comment (turnstep) [inline rebuild.sh]: This card is so large already, why don't we just store a copy of the patch itself?

File: bin/perlbug [hunk 3]
```diff
@@ -2,7 +2,7 @@
     eval 'exec /home/git/binary-com/perl/bin/perl -S $0 ${1+"$@"}'
 	if $running_under_some_shell;
 
-my $config_tag1 = '5.26.2 - Mon Jul 30 11:50:48 UTC 2018';
+my $config_tag1 = '5.26.2 - Mon Feb  3 08:21:42 UTC 2020';
```
Comment (turnstep) [inline bin/perlbug]: I've no idea how this all works, but why is the version and patchlevel_date staying the same, but the date changes?

File: lib/5.26.2/x86_64-linux/Config_heavy.pl [hunk 4]
```diff
@@ -1038,15 +1038,15 @@ sub myconfig {
...
+incpth='/usr/lib/gcc/x86_64-linux-gnu/6/include /usr/local/include /usr/lib/gcc/x86_64-linux-gnu/6/include-fixed /usr/include/x86_64-linux-gnu /usr/include'
 inews=''
 initialinstalllocation='/home/git/binary-com/perl/bin'
 installarchlib='/home/git/binary-com/perl/lib/5.26.2/x86_64-linux'
 installbin='/home/git/binary-com/perl/bin'
 installhtml1dir=''
 installhtml3dir=''
-installman1dir='/home/git/binary-com/perl/man/man1'
-installman3dir='/home/git/binary-com/perl/man/man3'
+installman1dir=''
```
Comment (turnstep) [inline lib/5.26.2/x86_64-linux/Config_heavy.pl:1048]: No man pages?

File: lib/5.26.2/x86_64-linux/Config_heavy.pl [hunk 5]
```diff
@@ -1122,12 +1122,12 @@ sub myconfig {
 mallocobj=''
 mallocsrc=''
 malloctype='void *'
-man1dir='/home/git/binary-com/perl/man/man1'
-man1direxp='/home/git/binary-com/perl/man/man1'
-man1ext='1'
-man3dir='/home/git/binary-com/perl/man/man3'
-man3direxp='/home/git/binary-com/perl/man/man3'
-man3ext='3perl'
+man1dir=' '
```
Comment (turnstep) [inline lib/5.26.2/x86_64-linux/Config_heavy.pl:1125]: s/{SPACE}// ?

File: rebuild.sh (hunk 2 above)
Comment (zakame) [reply rebuild.sh]: Yep thinking about that as well, probably better to keep it and maybe later patches in a directory to `cat | patch` through...

File: bin/perlbug (hunk 3 above)
Comment (zakame) [reply bin/perlbug]: This is filled in by the `make`s during build/install, presumably to let one know how recent this perl was configured.

File: lib/5.26.2/x86_64-linux/Config_heavy.pl (hunk 4 above)
Comment (zakame) [reply lib/5.26.2/x86_64-linux/Config_heavy.pl:1048]: Yep, we now have `-Dman1dir=none -Dman3dir=none` in our `./Configure` call.

File: rebuild.sh (hunk 2 above)
Comment (zakame) [reply rebuild.sh]: Now added as a local copy to `zcat | patch || exit 1` with.

File: lib/5.26.2/x86_64-linux/Config_heavy.pl (hunk 5 above)
Comment (zakame) [reply lib/5.26.2/x86_64-linux/Config_heavy.pl:1125]: Again another automatic `./Configure`/`make` change; we're due to update this anyway in a subsequent clean rebuild over a container.

File: lib/5.26.2/pod/perltoc.pod [hunk 6]
```diff
@@ -20144,146 +20144,6 @@ utf8, encode, compress, int32, null
 
 =back
 
-=head2 DB_File - Perl5 access to Berkeley DB version 1.x
```
Comment (zakame) [inline lib/5.26.2/pod/perltoc.pod]: Missed installing Berkeley DB (and GDBM) during the rebuild, these need to be here...

**Pull request merged**